core/src/main/java/tech/pegasys/web3signer/core/multikey/metadata/AbstractArtifactSignerFactory.java

```diff
@@ -106,6 +111,19 @@ protected Bytes extractBytesFromVault(final YubiHsm2SigningMetadata metadata) {
     }
   }
 
+  protected Bytes extractBytesFromInterlock(final InterlockSigningMetadata metadata) {
+    final InterlockSessionFactoryImpl interlockSessionFactory =
+        InterlockSessionFactoryProvider.newInstance(vertx, metadata.getKnownServersFile());
+    try (final InterlockSession session =
+        interlockSessionFactory.newSession(
```

Comment:
Think I remember that there can only be a single session existing at once, so this probably fail when we load multiple keys. As the SignerLoader processes the loading of keys in parallel.
Also isn't establishing a session expensive? So we only want to do this once for the all the interlock keys we load.